docs/user/boot_options.md

```diff
@@ -46,6 +46,15 @@ to extend.
   Setting this variable will impact all products.
   :::
 
+- `agama.finish`
+  During an unattended installation, if the installation is completed successfully then the
```

Comment:
so it does not have any effect if user do not specify agama.auto, right?

Reply:
Exactly, not by now.

Reply:
So may be mention that explicitly ... taking into account what bugs we sometimes get ;-)